<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>673</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_connectDB">
        <property name="text">
         <string>연결</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget_main">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_worker">
       <attribute name="title">
        <string>사람</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>목록</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_newHR">
              <property name="text">
               <string>추가</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_removeWorker">
              <property name="text">
               <string>삭제</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="listView_worker">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>등록정보</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>200</height>
               </size>
              </property>
              <property name="title">
               <string/>
              </property>
              <widget class="QGraphicsView" name="graphicsView">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>141</width>
                 <height>151</height>
                </rect>
               </property>
              </widget>
              <widget class="QLabel" name="label_workerName">
               <property name="geometry">
                <rect>
                 <x>180</x>
                 <y>10</y>
                 <width>400</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_workerRrNum">
               <property name="geometry">
                <rect>
                 <x>180</x>
                 <y>70</y>
                 <width>400</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_workerPhoneNum">
               <property name="geometry">
                <rect>
                 <x>180</x>
                 <y>100</y>
                 <width>400</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_workerAddr">
               <property name="geometry">
                <rect>
                 <x>180</x>
                 <y>130</y>
                 <width>400</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_workerMajor">
               <property name="geometry">
                <rect>
                 <x>180</x>
                 <y>40</y>
                 <width>400</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>업무내역</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_jobListForWorker">
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_company">
       <attribute name="title">
        <string>회사</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>목록</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_newCompany">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>추가</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_removeCompany">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>삭제</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="listView_company">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>정보</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QLabel" name="label_companyName">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>400</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_companyBlNum">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>400</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_companyOwner">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>400</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_companyAddress">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>130</y>
               <width>400</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_companyPhoneNum">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>100</y>
               <width>400</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_companyAccount">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>160</y>
               <width>400</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>업무내역</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_historyForCompany"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_work">
       <attribute name="title">
        <string>일</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QDateEdit" name="dateEdit_stat_from"/>
            </item>
            <item row="1" column="4">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <widget class="QDateEdit" name="dateEdit_stat_to"/>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_9">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>~</string>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QPushButton" name="pushButton_refreshJobjist">
              <property name="text">
               <string>적용</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>목록</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_workNew">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>추가</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_removeJobForStat">
              <property name="text">
               <string>삭제</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="tableView_jobListForStat">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::DotLine</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
